meta-xt-prod-devel-rcar-control/recipes-guests/domu/domu.bbappend

```diff
@@ -37,7 +38,11 @@ do_install:append() {
 
     if ${@bb.utils.contains('DISTRO_FEATURES', 'enable_virtio', 'true', 'false', d)}; then
         cat ${WORKDIR}/virtio.cfg >> ${CFG_FILE}
-        cat ${WORKDIR}/domu-vdevices-virtio.cfg >> ${CFG_FILE}
+        if ${@bb.utils.contains('MACHINE_FEATURES', 'gsx', 'true', 'false', d)}; then
+            cat ${WORKDIR}/domu-vdevices-virtio.cfg >> ${CFG_FILE}
```

Comment:
This is getting out of hand. We really need come .bbclass to generate those configs dynamically.

This is just a note... no actions need to be taken right now.

Reply:
Fully agree.